armory/baseline_models/keras/so2sat.py

```diff
@@ -66,5 +68,10 @@ def get_art_model(
     if weights_path:
         model.load_weights(weights_path)
 
-    wrapped_model = KerasClassifier(model=model, **wrapper_kwargs)
+    if "clip_values" in wrapper_kwargs:
+        if isinstance(wrapper_kwargs["clip_values"], list):
+            wrapper_kwargs2 = deepcopy(wrapper_kwargs)
```

Comment:
This should move outside the `if` statement, otherwise this will crash if `clip_values` is not set.  I can make this change.

Otherwise, LGTM